### ISSUE TITLE
DKIM and SPF documentation needed

### ISSUE THREAD

**Issue by jace** [hasgeek/hasjob#79]
Hasjob's listing confirmation emails may not be delivered if the domain's SPF and DKIM records aren't properly configured. This should be documented in the setup instructions and also fixed in the production installation. This bounce showed up in Sendgrid's logs today:

```
550 DKIM: encountered the following problem validating hasjob.co: pubkey_unavailable
```

**Comment by jace** [comment]
We've temporarily averted this by sending email from `@hasgeek.com`. @sindhus can you add this note to the documentation?

**Comment by sindhus** [comment]
What does DKIM and SPF stand for? 

Which section of documentation (am assuming you are referring to readme.md) do you want a line saying "Note: If the domain's DKIM and SPF when properly not confirgured result in Hasjob's listing confirmation emails not being delivered. We have temporarily averted this by sending emails from our domain `@hasgeek.com`"?

**Comment by jace** [comment]
Domain keys and Sender Policy Framework, two competing anti-spam methods. Hasjob sends a lot of email as a fundamental part of how it works. If it sends email from a domain without the appropriate DKIM and SPF settings, that email won't be delivered. The README should note this.

**Comment by jace** [comment]
Don't mention temporary workaround for HasGeek -- just a note that whatever domain is being used must have DKIM/SPF properly configured.

**Comment by pranshu792** [comment]
I think this would be provided as a solution by sendgrid, so you can close this ! :)

**Comment by jace** [comment]
Nope, Sendgrid can't provide this. The sending domain must have SPF and DKIM records explicitly provided.

**Comment by pranshu792** [comment]
Oh yes, agree with you .
